content/index.slim

```diff
@@ -4,16 +4,76 @@ title: Home
 .govuk-grid-row
   .govuk-grid-column-two-thirds
 
-    h2.govuk-heading-m
-      == link_to('Remote education during coronavirus (COVID-19)', '/teachers/')
+    #accordion-default.govuk-accordion[data-module="govuk-accordion"]
+      .govuk-accordion__section
+        .govuk-accordion__section-header
+          h2.govuk-accordion__section-heading
+            span#accordion-default-heading-1.govuk-accordion__section-button
+              | School curriculum and teaching
 
-    p.govuk-body
-      | Information, guidance and support for teachers and leaders on educating children during the coronavirus outbreak.
+        #accordion-default-content-1.govuk-accordion__section-content
+          ul.govuk-list
+            li = link_to("Supporting your children's education during coronavirus", "/supporting-your-childrens-education-during-coronavirus/").html_safe
+            li = link_to("Online education resources", "#").html_safe
+            li = link_to("Remote education during coronavirus", "/teachers").html_safe
```

Comment:
```suggestion
            li = link_to("Remote education during coronavirus", "/teachers/").html_safe
```
Think this might need a trailing slash for nginx

Reply:
:+1: 

Thanks!